src/config.rs

```diff
@@ -104,6 +104,39 @@ impl Config {
     pub fn new() -> Self {
         Self::default()
     }
+
+    pub fn increment_field(self: &mut Self, field: &str, value: i32) {
+        self.alter_numerical_field(field, value);
+    }
+
+    pub fn decrement_field(self: &mut Self, field: &str, value: i32) {
+        self.alter_numerical_field(field, -value);
+    }
+
+    fn alter_numerical_field(self: &mut Self, field: &str, value: i32) {
+        match field {
+            "app_bar_height" => self.app_bar_height += value,
+            "app_bar_bg" => self.app_bar_bg += value,
+            "app_bar_font_size" => self.app_bar_font_size += value,
+            "margin" => self.margin += value,
+            "padding" => self.padding += value,
+            _ => debug!("Attempt to alter unknown field: {} by {}", field, value)
```

Comment:
I feel like this should be more an `error!` than a `debug!`.
```suggestion
            _ => error!("Attempt to alter unknown field: {} by {}", field, value)
```